app/javascript/controllers/index.js

```diff
@@ -46,6 +48,10 @@ application.register('appointment-date', AppointmentDateController)
 application.register('autocomplete', AutocompleteController)
 application.register('collapse', CollapseController)
 application.register('conditional-field', ConditionalFieldController)
+application.register(
+  'confirm-item-accessories',
+  ConfirmItemAccessoriesController
+)
```

Comment:
I tried moving this back on its own line just cause it stands out and learned it's the linter that makes it go to multi-line. Silly linter. 😂